Small updates to the migration timeline wording (#13456)
DataDog/documentation: content/en/events/guides/migrating_to_new_events_features.md

## What Changed?

Please note that sending events does not change. You can continue sending events via the API, agent or using events via email feature as before. However, there are a number of changes to note.

### Event aggregations are no longer performed or displayed in the UI
Datadog no longer automatically performs Event aggregation, and no longer groups Events by the `aggregation_key` attribute. The UI no longer displays Event aggregation.

### Event comments are no longer supported or displayed in the UI
Comments created using the API with the `user_update` event type will be displayed as normal events.

> 

Author: ```suggestion
```

## What Changed?

> Please note that sending events does not change. You can continue sending events via the API, agent or using events via email feature as before. However, there are a number of changes to note.

Reviewer: ```suggestion
**Note:** The process for sending events remains the same. You can continue sending events using the API, the Agent, or the events via email feature as before.
```
